src/systems/ec/implementations/react/packages/card/stories/Template.jsx

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+
+import demoContentCard from '@ecl/ec-specs-card/demo/data--card';
+import demoContentTile from '@ecl/ec-specs-card/demo/data--tile';
+
+import Card from '../Card';
+
+const Template = () => {
+  const image1 = {
+    alt: demoContentCard.image.alt,
+    src:
+      'https://ec.europa.eu/commission/sites/beta-political/files/styles/header_image_breakpoints_theme_europa_wide_2x/public/homepage-banner-2400x900_003.jpg?yxX_LZza',
+  };
+  const image2 = {
+    alt: demoContentCard.image.alt,
+    src:
+      'https://ec.europa.eu/commission/sites/beta-political/files/styles/banner_image_breakpoints_theme_europa_normal_1x/public/parliament-plenary.png?itok=craeArBi&timestamp=1452594609',
+  };
+  const title1 = {
+    variant: 'standalone',
+    label: 'Policy coherence for development in the European Union',
+    href: '/example#1',
+    level: '3',
+  };
+  const title2 = {
+    variant: 'standalone',
+    label:
+      "The programming of EU's external assistance and development aid and the fragile balance of power bet ween EEAS and DG DEVCO",
+    href: '/example#2',
+    level: '3',
+  };
+  const title3 = {
+    variant: 'standalone',
+    label: 'Who gets what',
+    href: '/example#3',
+    level: '3',
+  };
+  const title4 = {
+    variant: 'standalone',
+    label:
+      'Tied aid and development aid procurement in the framework of EU and WTO law',
+    href: '',
+    level: '3',
+  };
+  const title5 = {
+    variant: 'standalone',
+    label: 'Cost of development policy',
+    href: '',
+    level: '3',
+  };
+  const title6 = {
+    variant: 'standalone',
+    label:
+      'France, Europe and development aid : from the treaties of Rome to the present day',
+    href: '',
+    level: '3',
+  };
+
+  return (
```

Comment:
If you're using an `ecl-row`, please wrap the template within an `ecl-container` and remove the `margin: 0` from the `ul`